<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICHCTimeFrame</class>
 <widget class="QFrame" name="ICHCTimeFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="horizontal2LineEdit"/>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_26">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Horizontal-1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="horizontalLineEdit"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Horizontal-2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_25">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="secondaryArmDownLineEdit"/>
   </item>
   <item row="0" column="4">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="secondaryArmUpLineEdit"/>
   </item>
   <item row="5" column="4">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="mainArmDownEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="vertical2LineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Vertical-2</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="mainArmBackwardEdit"/>
   </item>
   <item row="3" column="4">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="secondayGoBackwardLineEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="verticalineEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Vertical-1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="secondayGoForwardLineEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="mainArmUpEdit"/>
   </item>
   <item row="4" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="mainArmForwardEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
